test/cljs/reitit/frontend/core_test.cljs

```diff
@@ -99,21 +118,25 @@
                   :path-params {:id "5"}
                   :query-params {:mode "foo"}
                   :path "/5"
+                  :fragment nil
                   :parameters {:path {:id 5}
-                               :query {:mode :foo}}})
+                               :query {:mode :foo}
+                               :fragment nil}})
                (m (rf/match-by-path router "/5?mode=foo"))))
 
         (is (= "/5?mode=foo"
                (r/match->path (rf/match-by-name router ::foo {:id 5}) {:mode :foo}))))
 
-      (testing "fragment is ignored"
+      (testing "fragment string is read"
         (is (= (r/map->Match
                  {:template "/:id"
                   :path-params {:id "5"}
                   :query-params {:mode "foo"}
                   :path "/5"
+                  :fragment "fragment"
                   :parameters {:path {:id 5}
-                               :query {:mode :foo}}})
+                               :query {:mode :foo}
+                               :fragment "fragment"}})
```

Comment:
oh, I see. Like the `:body` in backend. Is this coherent to what we do with `:query` in case of not passing in value? e.g. what is emitted on `?kikka#kukka`

Reply:
No, the ring will consider query-string without `=` a single parameter without a value:
`{:query-params {:kikka ""} :fragment "kukka"}`